projects/charts-ng/package.json

```diff
@@ -20,7 +20,7 @@
   "peerDependencies": {
     "@angular/common": "20",
     "@angular/core": "20",
-    "echarts": "^5.6.0"
+    "echarts": "^5.6.0 || ^6.0.0"
```

Comment:
I think we should only allow 6 because with more upcoming changes like the containLabel and the new theming API, we need 6 only..